lib/active_model_serializers/callbacks.rb

```diff
@@ -0,0 +1,55 @@
+# Adapted from
+# https://github.com/rails/rails/blob/7f18ea14c8/activejob/lib/active_job/callbacks.rb
+require 'active_support/callbacks'
+
+module ActiveModelSerializers
+  # = ActiveModelSerializers Callbacks
+  #
+  # ActiveModelSerializers provides hooks during the life cycle of serialization and
+  # allow you to trigger logic. Available callbacks are:
+  #
+  # * <tt>around_render</tt>
+  #
+  module Callbacks
```

Comment:
You could... Docs just in the source for now... Copied out of activejob

B mobile phone

> On Nov 9, 2015, at 6:56 AM, Mauro George <notifications@github.com> wrote:
> 
> In lib/active_model_serializers/callbacks.rb:
> 
> > @@ -0,0 +1,55 @@
> > +# Adapted from
> > +# https://github.com/rails/rails/blob/7f18ea14c8/activejob/lib/active_job/callbacks.rb
> > +require 'active_support/callbacks'
> > +
> > +module ActiveModelSerializers
> > +  # = ActiveModelSerializers Callbacks
> > +  #
> > +  # ActiveModelSerializers provides hooks during the life cycle of serialization and
> > +  # allow you to trigger logic. Available callbacks are:
> > +  #
> > +  # * <tt>around_render</tt>
> > +  #
> > +  module Callbacks
> This callbacks is a public API? I mean I can use this on my PostSerializer for example? What is a use case for this? If this is the idea I think we need to add some documentation to it.
> 
> —
> Reply to this email directly or view it on GitHub.
>